docs/tutorials/getting_started/customize_your_deployment.md

```diff
@@ -91,8 +88,6 @@ By default, Data Docs are stored locally, in an uncommitted directory. This is g
 ## Additional Checkpoints and Actions
 Most teams will want to configure various Checkpoints and Validation Actions as part of their deployment. There are two primary patterns for deploying Checkpoints. Sometimes Checkpoints are executed during data processing (e.g. as a task within Airflow). From this vantage point, they can control program flow. Sometimes Checkpoints are executed against materialized data. Great Expectations supports both patterns. There are also some rare instances where you may want to validate data without using a Checkpoint.
 
-* [How to update Data Docs as a Validation Action](/docs/guides/validation/validation_actions/how_to_update_data_docs_as_a_validation_action)
```

Comment:
This exists 
https://docs.greatexpectations.io/docs/guides/validation/validation_actions/how_to_update_data_docs_as_a_validation_action#steps

Reply:
added back